.travis.yml

```diff
@@ -6,18 +6,25 @@ jdk:
 script:
   - mvn --settings .travis/settings.xml clean verify -B -V -DAWS_BUCKET=${AWS_BUCKET}
 
-before_deploy:
-  - echo $GPG_SECRET_KEY | base64 --decode | $GPG_EXECUTABLE --import
-  - echo $GPG_OWNERTRUST | base64 --decode | $GPG_EXECUTABLE --import-ownertrust
+#before_deploy:
+#  - echo $GPG_SECRET_KEY | base64 --decode | $GPG_EXECUTABLE --import
+#  - echo $GPG_OWNERTRUST | base64 --decode | $GPG_EXECUTABLE --import-ownertrust
 
 deploy:
-  -
-    skip_cleanup: true
-    provider: script
-    script: /bin/bash .travis/deploy.sh
+  - provider: openshift
+    user: $OPENSHIFT_USER
+    password: $OPENSHIFT_PASSWORD
+    domain: $OPENSHIFT_DOMAIN
+    app: datasafe-rest-service
     on:
-      tags: true
-      condition: "$TRAVIS_TAG =~ ^v([[:digit:]]+\\.)+[[:digit:]]+(-[[:digit:]]+)?(-SNAPSHOT)?$"
+      branch: feature/DOC-142_inbox_endpoints
+
+#  - provider: script
```

Comment:
Don't comment this, better extract to script that proceeds on some vairable

Reply:
ok